README.md

```diff
@@ -1,24 +1,24 @@
-Smart Bulb Gui Controller
+Smart Bulb GUI Controller
 =========================
 
-Contains a simple gui to control the power and brightness of Smart Bulbs. Currently only supports
+Contains a simple GUI to control the power and brightness of Smart Bulbs. Currently only supports
 Philips Hue (because this is what I own) - but the intention is to expand this to support any type
 of smart bulb available with an API.
 
-The next smart bulb to probably be supported is LifX
```

Comment:
"to probably be supported" splits the infinitive in an unclear way. Do you mean that it's likely that it's the next one to be supported, or that it's going to be supported in a probabilistic manner? 🚜